cypress/e2e/tests/pages/explorer/workloads/deployments.spec.ts

```diff
@@ -17,7 +17,7 @@ describe('Cluster Explorer', () => {
       deploymentsListPage = new WorkloadsDeploymentsListPagePo('local');
     });
 
-    describe('Create: Deployments', () => {
+    describe('Create: Deployments', { tags: ['@adminUser'] }, () => {
```

Comment:
The standard user should be able to create, update and delete resources in the `default` namespace (as they're a member of the `Default` project it's in)